modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -65,14 +65,15 @@ spec:
 <3> Specify a name for the output.
 <4> Specify the `elasticsearch` type.
 <5> Specify the URL and port of the external Elasticsearch instance as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
-<6> If using an `https` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
-<7> Optional: Specify a name for the pipeline.
-<8> Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
-<9> Specify the output to use with that pipeline for forwarding the logs.
-<10> Optional: Specify the `default` output to send the logs to the internal Elasticsearch instance.
-<11> Optional: Forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, OpenShift Logging removes the `structured` field and instead sends the log entry to the default index, `app-00000x`.
-<12> Optional: String. One or more labels to add to the logs.
-<13> Optional: Configure multiple outputs to forward logs to other external log aggregators of any supported type:
+<6> For a secure connection, you can specify an `https` or `http` URL that you authenticate by specifying a `secret`.
+<7> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. For more information, see the following "Example: Setting secret that contains a username and password."
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the output to use with that pipeline for forwarding the logs.
+<11> Optional: Specify the `default` output to send the logs to the internal Elasticsearch instance.
+<12> Optional: Forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, OpenShift Logging removes the `structured` field and instead sends the log entry to the default index, `app-00000x`.
```

Comment:
```suggestion
<12> Optional: Specify whether to forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, OpenShift Logging removes the `structured` field and instead sends the log entry to the default index, `app-00000x`.
```

I'm not sure if you want to maybe keep these consistent, i.e. use specify at the start of all lines?

Reply:
Updated. Thanks.